packages/svg-icons/README.md

```diff
@@ -10,6 +10,22 @@ npm install @fluentui/svg-icons
 
 ## Usage
 
+The library offers icons in SVG format; the icon names are structured as: 
+> `ic_fluent_[name]_[size]_[style]`
+
+* `name` - Name of the icon from [assets](../assets) that is all lowercased and underscore separated.
+* `size` - Size of the icon that is one of 16/20/24/28/48. Note that some icons do not have all sizes available yet. Our designers are working to add missing ones to complete the collection.
+* `style` - Style of the icon that is one of `regular`, `filled` or `selector`. See the section below for details.
```

Comment:
selector isn't an option in SVG...just Android

Reply:
Removed prefix and selector